.gitignore

```diff
@@ -2,7 +2,7 @@ bin
 debug
 build
 src/delegates.h
-
+env
```

Comment:
Where does this `env` come from?

Reply:
this is python env, as it is imperative to use venv while doing testing, i though its better to add it in .gitignore

Reply:
Oh, yes! This is a good idea.
Can you add a comment?

Reply:
Done. Also added __pycache__ which is generated during creation of environment.